src/store/app/selectors.ts

```diff
@@ -1,7 +1,7 @@
 import { useSelector } from 'react-redux';
-import { StoreState } from '..';
+// import { StoreState } from '..';
 import type { AppState } from '../types';
 
-export const selectApp = (): AppState => {
-  return useSelector<StoreState, AppState>(state => state.app);
-};
+// export const selectApp = (): AppState => {
+//   return useSelector<StoreState, AppState>(state => state.app);
+// };
```

Comment:
Why do you commented it?

Reply:
I have an error with StoreState import there, but now it's gone. Maybe it was lag of my VS Code. I returned everything back.